src/main/java/thymeleaf/ThymeleafViewRenderer.java

```diff
@@ -0,0 +1,63 @@
+package thymeleaf;
+
+import io.dropwizard.views.View;
+import io.dropwizard.views.ViewRenderer;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.templateresolver.TemplateResolver;
+
+import javax.ws.rs.WebApplicationException;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.util.Locale;
+import java.util.Map;
+
+public class ThymeleafViewRenderer implements ViewRenderer {
+
+    private final TemplateEngine engine;
+    private final String suffix;
+
+    public ThymeleafViewRenderer(
+            String templateMode,
+            String prefix,
+            String suffix,
+            boolean cacheable) {
+        super();
+
+        this.suffix = suffix;
+
+        TemplateResolver templateResolver = new TemplateResolver();
+        templateResolver.setResourceResolver(new ThymeleafResourceResolver());
+        templateResolver.setTemplateMode(templateMode);
+        templateResolver.setPrefix(prefix);
+        templateResolver.setCacheable(cacheable);
+        engine = new TemplateEngine();
+        engine.setTemplateResolver(templateResolver);
+
+    }
+
+    @Override
+    public boolean isRenderable(View view) {
+        return view.getTemplateName().endsWith(suffix);
```

Comment:
shouldn't this check for `instanceof ThymeleafView` instead of doing a string comparison?